doc/api/url.md

```diff
@@ -234,6 +234,16 @@ url.resolve('/one/two/three', 'four')         // '/one/two/four'
 url.resolve('http://example.com/', '/one')    // 'http://example.com/one'
 url.resolve('http://example.com/one', '/two') // 'http://example.com/two'
 ```
+Note that `url.resolve()` method behaves differently depending on the protocol given in the `from` parameter. Some protocols like https, file, ftp, wss and gopher have special cases.
```

Comment:
A couple of suggestions:

- Can you leave a blank line above this one?
- Can you try to wrap the line after 80 characters?
- I’d suggest enclosing the protocol names in backticks, i.e. `` `https` ``, that seems a bit more readable to me.